src/clusterfuzz/_internal/metrics/monitoring_metrics.py

```diff
@@ -32,6 +32,19 @@
         monitor.BooleanField('bad_build'),
     ])
 
+JOB_BUILD_RETRIEVAL_TIME = monitor.CumulativeDistributionMetric(
+    'task/fuzz/job/build_retrieval_time',
+    bucketer=monitor.FixedWidthBucketer(width=0.05, num_finite_buckets=20),
+    description=('Distribution of fuzz task\'s build retrieval times. '
```

Comment:
This metric name is misleading, this instrumentation applies for all tasks that invoke build_manager.setup_build, fix it.